platform/android/export/export.cpp

```diff
@@ -198,6 +198,80 @@ static const char *android_perms[] = {
 	NULL
 };
 
+//TODO: put this somewhere better (i.e. templates folder)
+const String ANDROID_MANIFEST_TEXT = "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n"
+									 "<manifest xmlns:android=\"http://schemas.android.com/apk/res/android\"\n"
```

Comment:
Running clang formatting gives me this super huge spacing

Reply:
What if line 202 looks like this? Same?

```
const String ANDROID_MANIFEST_TEXT =
  "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n"
```